qiita_db/metadata_template/sample_template.py

```diff
@@ -57,13 +56,12 @@ class SampleTemplate(MetadataTemplate):
     """
     _table = "study_sample"
     _table_prefix = "sample_"
-    _column_table = "study_sample_columns"
     _id_column = "study_id"
     _sample_cls = Sample
     _filepath_table = 'sample_template_filepath'
 
-    @staticmethod
-    def metadata_headers():
+    @classmethod
+    def metadata_headers(cls):
```

Comment:
Should this method be moved to the base class so it also becomes available for the PrepTemplate class?